realm/realm-annotations-processor/src/main/java/io/realm/processor/Constants.java

```diff
@@ -43,29 +43,41 @@
      * Realm types and their corresponding Java types
      */
     public enum RealmFieldType {
-        NOTYPE(null, "Void"),
-        INTEGER("INTEGER", "Long"),
-        FLOAT("FLOAT", "Float"),
-        DOUBLE("DOUBLE", "Double"),
-        BOOLEAN("BOOLEAN", "Boolean"),
-        STRING("STRING", "String"),
-        DATE("DATE", "Date"),
-        BINARY("BINARY", "BinaryByteArray"),
-        REALM_INTEGER("INTEGER", "Long"),
-        OBJECT("OBJECT", "Object"),
-        LIST("LIST", "List"),
-        BACKLINK("BACKLINK", null);
+        NOTYPE(null, "Void", false),
+        INTEGER("INTEGER", "Long", false),
+        FLOAT("FLOAT", "Float", false),
+        DOUBLE("DOUBLE", "Double", false),
+        BOOLEAN("BOOLEAN", "Boolean", false),
+        STRING("STRING", "String", false),
+        DATE("DATE", "Date", false),
+        BINARY("BINARY", "BinaryByteArray", false),
+        REALM_INTEGER("INTEGER", "Long", false),
+        OBJECT("OBJECT", "Object", false),
+        LIST("LIST", "List", true),
+
+        BACKLINK("LINKING_OBJECTS", null, true),
```

Comment:
will this result https://github.com/realm/realm-java/pull/5209/files#diff-36f5b86acab52385e590ad92a6d7a80cR690 add backlink `RealmResults` as a persisted property? 
Would `isList()` useful for backlinks? maybe change it to `isRealmList` and return false for backlink if it is not useful for it?